docker/api/src/Domain/User/User.php

```diff
@@ -190,9 +193,14 @@ public function verifyPassword(string $password): bool{
         return password_verify($password, $this->password_hash);
     }
 
-    #[Pure] #[ArrayShape(['userId' => "int", 'firstName' => "string", 'lastName' => "string", 'email' => "string", 'balance' => "string", 'shoppingCart' => 'array', 'address' => "string",'city' => "string",'state' => "string",'zip' => "string",'country' => "string"])]
+    #[Pure] #[ArrayShape(['userId' => "int", 'firstName' => "string", 'lastName' => "string", 'email' => "string", 'balance' => "string", 'shoppingCart' => 'array', 'address' => "string",'city' => "string",'state' => "string",'zip' => "string",'country' => "string", 'profileImage' => 'string'])]
     public function jsonSerialize(): array{
 
+        //Profile Image
+        $userId = $_COOKIE['freshMartUserId'];
```

Comment:
On the Checkout page this line is causing the following error response:
`"ERROR: Undefined array key \"freshMartUserId\" on line 200 in file \/var\/www\/html\/api\/src\/Domain\/User\/User.php."`
Can you check if you are getting this as well?

Reply:
I didn't consider  non-authenticated requests when I wrote that.  It now checks to make sure the cookie is set first.